core/src/commonMain/kotlin/com/lightspark/sdk/core/crypto/SecureRandom.kt

```diff
@@ -0,0 +1,3 @@
+package com.lightspark.sdk.core.crypto
```

Comment:
Let's use an `@file:JvmName` annotation to make sure this doesn't have "Kt" at the end of it when called from Java.